bindings/pydrake/multibody/all.py

```diff
@@ -3,14 +3,6 @@
 # Deprecated symbols.
 with warnings.catch_warnings():
     warnings.simplefilter("ignore", DeprecationWarning)
-    # - Attic (#9314)
-    from .collision import *
-    from .joints import *
-    from .parsers import *
-    from .rigid_body_plant import *
-    from .rigid_body_tree import *
-    from .rigid_body import *
```

Comment:
Note: this change torpedoed my underactuated notes (there was no warning on that side suggesting these would go away)

https://circleci.com/gh/RussTedrake/underactuated/1193?utm_campaign=workflow-failed&utm_medium=email&utm_source=notification